tests/test_vpc.py

```diff
@@ -8,6 +8,18 @@
 from botocore.exceptions import ClientError as BotoClientError
 from c7n.exceptions import PolicyValidationError
 from c7n.resources.aws import shape_validate
+from pytest_terraform import terraform
+
+
+@terraform('sg')
+def test_codebuild_unused(test, sg):
+    factory = test.replay_flight_data("test_security_group_codebuild_unused")
+    p = test.load_policy(
+        {"name": "sg-unused", "resource": "security-group", "filters": ["unused"]},
+        session_factory=factory,
+    )
+    resources = p.run()
```

Comment:
this causes a good amount of api calls that are mostly extraneous, either manually removing the content, or patching the resource to just do the codebuild check, or just exercising that method on the filter would result in significantly less recorded content.

Reply:
Patched it so that only `get_codebuild_sgs` is called.